docs/millennium-os/manual/chapters/30_usage.md

```diff
@@ -43,22 +45,22 @@ flowchart TB
 ```
 
 !!! warning
-    You can clearly see here the "mismatch" between Fusion360 and RRF / DWC - **Work Offset 0** in Fusion360 corresponds to **WCS 1** in DWC (the offset is *from the first WCS* rather than from *G53, or Machine Co-ordinates*). This is because we do not allow code in machine co-ordinates to be outputted.
+    You can see here the "mismatch" between Fusion360 and RRF / DWC - **Work Offset 0** in Fusion360 corresponds to **WCS 1** in DWC (the offset is *from the first WCS* rather than from *G53, or Machine Co-ordinates*). This is because we do not allow code in machine coordinates to be outputted.
 
-    You need to remember this when setting your work offset value in Fusion360 or setting your WCS origins manually.
+    You must remember this when setting your work offset value in Fusion360 or setting your WCS origins manually.
```

Comment:
```suggestion
```

This whole warning block can be removed as we've changed how the offset system works with `v0.3.0`.